modules/codegen/src/smithy4s/codegen/PathRef.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright 2021-2024 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s.codegen
+
+final case class PathRef(underlying: os.Path)
```

Comment:
this should not be part of the `codegen` module, but rather the sbt module.

Reply:
Sure, will move it

Reply:
acfd9ddcef09095f154acdb34abe202543da5483